lang/syn/src/parser/accounts/constraints.rs

```diff
@@ -291,6 +291,18 @@ pub fn parse_token(stream: ParseStream) -> ParseResult<ConstraintToken> {
                         error: parse_optional_custom_error(&stream)?,
                     },
                 )),
+                "realloc" => ConstraintToken::Realloc(Context::new(
+                    span,
+                    ConstraintRealloc {
+                        space: stream.parse()?,
+                    },
+                )),
+                "allocator" => ConstraintToken::Allocator(Context::new(
```

Comment:
I wonder if we can do something a little more clear. When I see `allocator` I think of a memory allocator. Perhaps `realloc::payer` or something else would be more descriptive?

Reply:
yeah i tried to think of a word that would indicate a combination of payer and receiver depending on the direction of lamport transfer